docs/cli.rst

```diff
@@ -107,7 +107,7 @@ You can also specify the location yourself using the :option:`--config` option.
 
 .. note::
 
-  On Windows there is a default config created by the installer but on any
+  On Windows, there is a default config created by the installer, but on any
```

Comment:
I don't think the commas are needed here.

Reply:
No, both commas are correct here. The first one comes after the introductory and the second one separates an independent clause.

Reply:
I believe English is taught better in non-english speaking countries... 😞

Reply:
@beardypig I'm a native English speaker and while grammatically it may be correct it sounds odd when read aloud. English is a garbage language.

Reply:
@gravyboat as am I and i agree with everything you said.